### DISCUSSION TITLE
Newbie Question - Missing Icons in Courseplay?

### DISCUSSION THREAD

**Discussion by Keylockman8** [Courseplay/Courseplay_FS22#1721] [Q&A]
New here; upfront apologies if not posted in the correct space.

Newbie question (actually I have watched a few Courseplay tutorials, and had previously loaded and activated both Courseplay and Autodrive in my saved game (Fruhling), but never used) - I want to learn to use Courseplay and Autodrive.  So after several weeks of inactivity, I am starting over.  I have downloaded and activated the latest versions (7.1.0.0 CP and 2.0.0.6 AD) of both.  But I no longer see all three CP icons; I see the CP Global Settings, and can open that, but I do not see either the CP Vehicle Settings or the Course Management Setting.  Tried on both Fruhling saved game (deactivated all mods except CP and AD), as well as a new game on Calmsden.  Same thing.  Is something wrong with my game / gameplay, or are these Settings now elsewhere?

**Comment by Tensuko** [comment]
The vehicle setting and course setting only comes up when you are inside a vehicle or selected one on the AI Map.

**Reply by Keylockman8** [reply]
Thank you, Tensuko.  I just returned home and was able to check this and you are correct; I was not in a vehicle when checking.  I will keep plugging away and hopefully not have any problems.  Much appreciated!
